src/com/sun/ts/tests/servlet/api/javax_servlet/genericfilter/InitFilterConfigTestServlet.java

```diff
@@ -2,7 +2,7 @@
 *
 * The Apache Software License, Version 1.1
 *
-* Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+* Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
The changes on this file do not look needed.

Reply:
Fixed.